.github/workflows/docker-image.yml

```diff
@@ -10,6 +10,17 @@ jobs:
     steps:
       - name: ☁️ Checkout source
         uses: actions/checkout@v3
+      - name: Docker metadata action
+        id: meta_backend
+        uses: docker/metadata-action@v4.1.1
+        with:
+          images: |
+            infisical/backend
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
```

Comment:
I just read the action documentation, and IMHO, labeling the images with `PR-n` is not something I would advise.